modules/messages/src/instant_payments.rs

```diff
@@ -85,16 +88,38 @@ where
 	}
 
 	fn pay_relayers_rewards(
+		lane_id: LaneId,
+		messages_relayers: VecDeque<UnrewardedRelayer<T::AccountId>>,
 		confirmation_relayer: &T::AccountId,
-		relayers_rewards: RelayersRewards<T::AccountId, Currency::Balance>,
+		received_range: &RangeInclusive<u64>,
 		relayer_fund_account: &T::AccountId,
-	) {
-		pay_relayers_rewards::<Currency, _>(
-			confirmation_relayer,
-			relayers_rewards,
-			relayer_fund_account,
-			GetConfirmationFee::get(),
-		);
+	) -> Result<(), Self::Error> {
```

Comment:
Is there any specific reason to return `Result` now? I mean - the outside function would just ignore that error, because storage has already been altered && we don't rollback these changes.

Reply:
No extra specific reason now. I wonder if we can add the `[transactional]` tag to the message pallet dispatch call. So this return error would be useful.